snorkel/types/logger.py

```diff
@@ -0,0 +1,5 @@
+from typing import Any, Dict
+
+# A config dict is a nested dict with a max depth of 3
+Config = Dict[str, Any]
+Path = str
```

Comment:
this might be confusing to people since `Path` objects exist elsewhere. can we just use str in-place?

Reply:
Yep, will remove the `Path` type for now, though in a future commit I suggest we call-out where strings are paths explicitly.